### PR TITLE
[Spring Beans] LPS-201364

### PR DESCRIPTION
**Pull request by JanisZhang**
Move `PDFMetadataProcessorMessageListener`, `VideoProcessorMessageListener`, `AudioProcessorMessageListener`, `ImageProcessorMessageListener` to the module and register Destination in the activate method. Thanks.
https://liferay.atlassian.net/browse/LPS-201364

### CODE REVIEW

**Comment by JanisZhang** [comment]
Trigger CI tests as https://github.com/liferay-core-infra/liferay-portal/pull/6058 passed review.

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by JanisZhang** [comment]
ci:test:document

File: modules/apps/document-library/document-library-preview-video/src/main/java/com/liferay/document/library/preview/video/internal/messaging/VideoProcessorMessageListener.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+@Component(
+	property = "destination.name=" + DestinationNames.DOCUMENT_LIBRARY_VIDEO_PROCESSOR,
+	service = MessageListener.class
+)
+public class VideoProcessorMessageListener
+	extends BaseProcessorMessageListener {
+
+	@Activate
+	protected void activate(BundleContext bundleContext) {
+		DestinationConfiguration destinationConfiguration =
```
Comment (dantewang) [inline modules/apps/document-library/document-library-preview-video/src/main/java/com/liferay/document/library/preview/video/internal/messaging/VideoProcessorMessageListener.java:38]: @JanisZhang Also this one..

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by JanisZhang** [comment]
@dantewang Force-pushed. I only inline the variable. Thanks!

**Review by dantewang — APPROVED** [review]

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by JanisZhang** [comment]
ci:test:document

**Comment by dantewang** [comment]
@zhangyan0701 Could you review the test results? Thanks!

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by dantewang** [comment]
ci:forward:force

**Pull request closed without merging**